src/core/packages/saved-objects/api-server-internal/src/lib/apis/create.ts

```diff
@@ -159,7 +159,7 @@ export const performCreate = async <T>(
     refresh,
     document: raw._source,
     ...(overwrite && version ? decodeRequestVersion(version) : {}),
-    require_alias: true,
+    querystring: { require_alias: true },
```

Comment:
`require_alias` is accepted by the `IndexRequest` but not the `CreateRequest`, so I went with the `querystring` passthrough for both here.